packages/api/src/semantic-relationship/index.ts

```
@@ -58,7 +58,12 @@ export default class SemanticRelationship extends Ad4mModel {
         ${SEMANTIC_RELATIONSHIP_FOR_ITEM.replace("ItemId", `"${itemId}"`)}
         ${EMBEDDING_FROM_SEMANTIC_RELATIONSHIP}.
       `);
-      return result[0]?.Embedding ? JSON.parse(result[0].Embedding) : [];
+
+      if (!result?.[0]?.Embedding) return []
+      else {
+        const embeddingExpression = await this.perspective.getExpression(result[0].Embedding);
+        return JSON.parse(embeddingExpression.data);
+      };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)